src/remotestorage.ts

```diff
@@ -257,7 +257,17 @@ class RemoteStorage {
       options.scope = this.access.scopeParameter;
     }
 
-    options.redirectUri = globalContext.cordova ? config.cordovaRedirectUri : String(Authorize.getLocation());
+    if (globalContext.cordova) {
+      options.redirectUri = config.cordovaRedirectUri;
+    } else {
+      let location = Authorize.getLocation();
```

Comment:
I think this should be `const`, as it's not redefined, no?

Reply:
Right.